pkg/sql/execinfrapb/processors_bulk_io.proto

```diff
@@ -282,11 +293,15 @@ message CSVWriterSpec {
   // User who initiated the export. This is used to check access privileges
   // when using FileTable ExternalStorage.
   optional string user_proto = 6 [(gogoproto.nullable) = false, (gogoproto.casttype) = "github.com/cockroachdb/cockroach/pkg/security.SQLUsernameProto"];
+
+  // col_names specifies the logical column names for the exported parquet file.
+  repeated string col_names = 9 ;
 }
 
 // ParquetWriterSpec is the specification for a processor that consumes rows and
 // writes them to Parquet files at uri. It outputs a row per file written with
 // the file name, row count and byte size.
+// DEPRECATE
 message ParquetWriterSpec {
```

Comment:
Is it safe to deprecate this proto message?

Reply:
I believe we have never shipped a version with this proto message defined so we should be safe to remove it completely.

Reply:
Done